mijia-homie/src/main.rs

```diff
@@ -474,6 +657,50 @@ async fn service_bluetooth_event_queue(
 }
 
 async fn handle_bluetooth_event(
```

Comment:
```suggestion
async fn handle_broadcast_event(
```

Reply:
```suggestion
async fn handle_advertisement_event(
```

Reply:
I called it `handle_bluetooth_event` because it handles a `BluetoothEvent`, i.e. directly from `bluez-async`, as opposed to a `MijiaEvent`. It could handle things other than advertisements in future.

Reply:
I've refactored to handle all bluetooth events here, including converting to `MijiaEvent` if appropriate. See what you think.